packages/twenty-server/src/core/api-rest/api-rest-query-builder/utils/parse-path.utils.ts

```diff
@@ -19,3 +19,47 @@ export const parsePath = (
 
   return { object: queryAction[0], id: queryAction[1] };
 };
+
+export const parseMetadataPath = (
+  request: Request,
+): { object: string; id?: string } => {
+  const queryAction = request.path.replace('/rest/metadata/', '').split('/');
+
+  if (queryAction.length > 3) {
+    throw new BadRequestException(
+      `Query path '${request.path}' invalid. Valid examples: /rest/metadata/fields or /rest/metadata/objects/id`,
+    );
+  }
+
+  if (!['fields', 'objects', 'relations'].includes(queryAction[0])) {
```

Comment:
Would there be a way to somehow fetch these values directly from the db ? I don't feel comfortable hardcoding it. Or maybe I am missing something and we shouldn't do it like this

Reply:
looks like hardcoding it is the best we can do for now